dts/bindings/gpio/nxp,lpc-gpio.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2019, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: LPC GPIO
+version: 0.1
+
+description: >
+    This is a representation of the LPC GPIO nodes
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "nxp,lpc-gpio"
+      generation: define
```

Comment:
Include base.yaml and remove the properties that are now defined there.

Reply:
@MaureenHelm I revisited this pull yesterday, and realized after a rebase that I need to do some rework on this pull, including this .yaml.

Reply:
@MaureenHelm i've reworked this pull, please review it again.